addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -4034,6 +4034,35 @@ X[]
                     });
                 });
             });
+            describe("Links", () => {
```

Comment:
We are testing ```rolt="tab"``` so string "Links" does not make sense here.

Reply:
Yes but its the link `a` with different roles. How about `Link with different roles` ?